CHANGELOG.md

```diff
@@ -3,6 +3,58 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+
+## v5.11.1
+
+_May 29, 2022_
+
+We'd like to offer a big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Support localization on the date and time picker components (#4517) @alexfauquette
```

Comment:
Putting a working code will take a lot of space, maybe a link to the doc is enough. If in the next release we bring full support of localization, it will be time to ask for help in translation aria texts

```suggestion
- 🌍 Support localization on the date and time picker components (#4517) @alexfauquette

	You can translate some texts as you do with the data grid. Check the [localization documentation for more information](mui.com/x/react-date-pickers/localization/)
```